test/new-e2e/go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/DataDog/datadog-agent/pkg/util/pointer v0.48.1
 	github.com/DataDog/datadog-api-client-go v1.16.0
 	github.com/DataDog/datadog-api-client-go/v2 v2.15.0
-	github.com/DataDog/test-infra-definitions v0.0.0-20231027164033-8158a5b2c60e
+	github.com/DataDog/test-infra-definitions v0.0.0-20231103172917-b3e53c405861
```

Comment:
You should also update `TEST_INFRA_DEFINITIONS_BUILDIMAGES` in `.gitlab-ci.yml` according to the comment above https://github.com/DataDog/datadog-agent/pull/20618/files#diff-fee3041315eef2d6446d1e0f565e42851503923fb40e53cf4c91ac5641ebc4e0R13

Reply:
Indeed !
It is now fixed.